mandatory/1-syntax-errors.js

```diff
@@ -30,8 +31,8 @@ test("addNumbers adds numbers correctly", () => {
 });
 
 test("introduceMe function returns the correct string", () => {
-  expect(introduceMe("Sonjide", 27)).toEqual(
-    "Hello, my name is Sonjide and I am 27 years old"
+  expect(introduceMe("Sonjide", 28)).toEqual(
+    "Hello, my name is Sonjide and I am 28 years old"
```

Comment:
Hi Danny, in exercies, you don't need to change the tests scripts. If your function is written correctly, the tests functions test them by sending some random inputs and you can see if your function is written correct or not. :)